pkg/agent/utils.go

```diff
@@ -569,43 +569,3 @@ func addFeatureGateString(featureGates string, key string, value bool) string {
 	}
 	return strings.Join(pairs, ",")
 }
-
-// ParseCSEMessage parses the raw CSE output
-func ParseCSEMessage(message string) (*datamodel.CSEStatus, *datamodel.CSEStatusParsingError) {
```

Comment:
we should merge the RP side first right? or at least before vendor?

Reply:
yep exactly

Reply:
RP side was merged, this should be good to go I think